Userland/Libraries/LibGfx/Font/OpenType/Tables.h

```diff
@@ -171,6 +171,36 @@ struct GlyphHorizontalMetrics {
     i16 left_side_bearing;
 };
 
+// https://learn.microsoft.com/en-us/typography/opentype/spec/fpgm
+// fpgm: Font Program
+struct Fpgm {
```

Comment:
Reading fpgm/prep in a commit about adding accessors? rebase mixup perhaps?

Reply:
Ah, I see where @ldm5180 was coming from now

Reply:
(Split this out into two commits now)